UPGRADE-4.0.md

```diff
@@ -24,6 +24,9 @@ Serializer
 Yaml
 ----
 
+ * The `Dumper::setIndentation()` method was removed. Pass the indentation
+   level to the constructor instead.
```

Comment:
I'm only worried that we document that something was removed in UPGRADE-4.0.md file, but then we forget to remove it.